docs/jazzy.yml

```diff
@@ -159,6 +174,14 @@ custom_categories:
       - RoadClosureSubtype
       - FeedbackSource
       - EndOfRouteFeedback
+      - FeedbackEvent
```

Comment:
Decided to also add other missing symbols. There is quite a lot of misalignment at this point, will do further clean-up in subsequent PRs.